custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html

```diff
@@ -1,24 +1,10 @@
 {% load staticfiles %}
 <div class="app-bg min-vh-100">
   <div ng-show="!$ctrl.filtersOpen">
+    <div class="top-section p-3">
+      {% include 'icds_reports/icds_app/mobile/partials/mobile-heading.html' %}
+    </div>
```

Comment:
this is more than just a refactor - it changes how navigation works on map / chart pages. addresses this card: https://trello.com/c/YE5QGErC/239-decide-on-final-navigation-plan